README.md

```diff
@@ -102,6 +104,40 @@ rdfVocabularies({ only: ['rdfs', 'owl', 'skos'] })
   })
 ```
 
+#### Getting a Readable Stream (Quad Stream)
+
+```js
+const rdfVocabularies = require('@zazuko/rdf-vocabularies')
+const stream = await rdfVocabularies({ stream: true, only: ['rdfs', 'owl', 'skos'] })
+```
+
+### Expanding a Prefix
+
+There are two ways of expanding a prefix:
+
+* `rdfVocabularies.expand(prefixedTerm): String` synchronous
+
+    Expand without checks. It is similar to prefix.cc in the sense that prefix.cc would expand
+    `schema:ImNotInSchemaDotOrg` to `http://schema.org/ImNotInSchemaDotOrg`.
+
+* `rdfVocabularies.expand(prefixedTerm, Array<String>): Promise<String>` **asynchronous**
+
+    Expand with type check:
+    ```js
+    const rdfVocabularies = require('@zazuko/rdf-vocabularies')
+    const Class = rdfVocabularies.expand('rdfs:Class')
+    const Property = rdfVocabularies.expand('rdf:Property')
+
+    // Will return `http://schema.org/Person` iff the dataset contains either
+    // <schema:Person> <rdf:type> <rdfs:Class>
+    // or
+    // <schema:Person> <rdf:type> <rdf:Property>
```

Comment:
Only class and property? What about datatypes?

Reply:
I never used those. Any suggestion on how to make this API accommodate them?

Reply:
I think it would be enough to also look for `?t <rdf:type> <rdfs:Datatype>`

Reply:
Yep, you could do this by passing your term to the 2nd argument: `await rdfVocabularies.expand('schema:Person', [Class, Property])`